tools/topology/topology1/m4/multiband_drc_coef_default.m4

```diff
@@ -1,7 +1,7 @@
-# Exported Control Bytes 11-May-2021
+# Exported Control Bytes 16-Aug-2023
```

Comment:
Whats the command line to reproduce ?

Reply:
Seems I need to update #8142 to update this comment for tplg1 too. This change is already merged to main so I'd need to update it there as well. But I don't think the setup tool update is worth to do for v2.2. It would need to be a new middle version of v2.2 and git main without tplg2 part.

The tplg2 is already in order. This tplg1 part was split out from the tplg2 patch.